libnemo-private/nemo-global-preferences.h

```diff
@@ -74,6 +74,9 @@ typedef enum
 /* Activating executable text files */
 #define NEMO_PREFERENCES_EXECUTABLE_TEXT_ACTIVATION		"executable-text-activation"
 
+/* Image viewers to pass nemo view sort order to */
+#define NEMO_PREFERENCES_IMAGE_VIEWERS_WITH_EXTERNAL_SORT "image-viewers-with-external-sort"
```

Comment:
It seemed useful to have this controllable by property, but felt a bit low level to be referencing executable on the preferences page, so it's a hidden property for now. This could be supplemented with an extra toggle property to enable it for any in this list.

Reply:
This works - we can add more as they're discovered.

I don't see any reason to be able enable/disable this behavior - it's ok as the 'normal' behavior I think. It doesn't interfere with the single selection behavior (browsing the entire folder).